lib/topological_inventory/openshift/operations/worker.rb

```diff
@@ -61,6 +61,7 @@ def order_service(client, msg)
             service_plan.name, service_offering.name, order_params
           )
           client.ack(msg.ack_ref)
+          catalog_client.wait_for_provision_complete(service_instance)
```

Comment:
It seems this can be a long running process in thread. It then blocks other messages from the queue.

Reply:
Yes, we still need to handle processing this in another thread which @slemrmartin has another PR for